### ISSUE TITLE
Data entry users should not be able to remove attendees

### ISSUE THREAD

**Issue by cecilia-donnelly** [OpenTechStrategies/swoptact#267]
A user requests a permission level change:

>1) As a person entering information under a leader (DataEntry)  I can delete participants under the Attendees by selecting the X button – can this button be isolated for Data Entry people (aka Leaders) so that they do not have the option to delete?  Please see attachment for figure. 

On the "Action" page, in the sign-in sheet section, only admins and staff should be able to remove attendees.  Currently all users are able to remove attendees.

**Comment by cecilia-donnelly** [comment]
This should be relatively straightforward to fix.  In `inline_ajax.js`, look at the `userCanEdit()` function.  It's currently called in two places: once to create the delete button and the other to make rows editable.  We now need to split that into `userCanEdit()` and `userCanDelete()` so that data entry users (leaders) can have permission to edit but not delete (or unlink).

**Comment by kfogel** [comment]
I was able to reproduce this.  Logged in as a user "reguser" who has only the Leader role, I went to http://localhost:8000/swoptact/event/8/, the "Local Baseball Tournament" event in our sample data.  The sign-in sheet had 8 people, and in the rightmost column of each person's row was a red Delete X-in-a-circle button.  When I clicked that button for user Eustace Solares, the user's row disappeared.  (I then found myself wishing for an undo button, as per issue #188, but whatever -- it's just sample data; I can reload it any time.)

So, that's the bug.  Reproduced -- now to fix.
